packages/emails/src/components/ManageLink.tsx

```diff
@@ -4,27 +4,29 @@ import type { CalendarEvent, Person } from "@calcom/types/Calendar";
 export function ManageLink(props: { calEvent: CalendarEvent; attendee: Person }) {
   // Only the original attendee can make changes to the event
   // Guests cannot
-  if (props.attendee.email !== props.calEvent.attendees[0].email) return null;
   const t = props.attendee.language.translate;
-
-  return (
-    <div
-      style={{
-        fontFamily: "Roboto, Helvetica, sans-serif",
-        fontSize: "16px",
-        fontWeight: 500,
-        lineHeight: "0px",
-        textAlign: "left",
-        color: "#3e3e3e",
-      }}>
-      <p>
-        <>{t("need_to_reschedule_or_cancel")}</>
-      </p>
-      <p style={{ fontWeight: 400, lineHeight: "24px" }}>
-        <a href={getCancelLink(props.calEvent)} style={{ color: "#3e3e3e" }}>
-          <>{t("manage_this_event")}</>
-        </a>
-      </p>
-    </div>
-  );
+  if(props.attendee.email === props.calEvent.attendees[0].email || props.calEvent.organizer.email  === props.attendee.email ){
```

Comment:
I switch this if statement cause I couldn't for the life of me to get it to work and I'm not sure why :S

Reply:
Let's figure out why and take the learning instead of just patching